mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -459,6 +459,42 @@ pub enum TendermintCoinRpcError {
     NotFound(String),
 }
 
+#[derive(Display, Debug)]
+pub enum IBCChannelError {
+    #[display(
+        fmt = "IBC channel could not be found in coins file for '{}' address prefix. Provide it manually by including `ibc_source_channel` in the request.",
+        address_prefix
+    )]
+    IBCChannelCouldNotFound { address_prefix: String },
```

Comment:
`IBCChannelCouldNotFound` should be `IBCChannelNotFound` or `IBCChannelCouldNotBeFound`